homekit/controller.py

```diff
@@ -280,6 +281,22 @@ def list_accessories_and_characteristics(self):
             raise
         tmp = response.read().decode()
         accessories = json.loads(tmp)['accessories']
+        
+        for accessory in accessories:
+            for service in accessory['services']:
+                service['type'] = service['type'].upper()
+                try:
+                    service['type'] = ServicesTypes.get_uuid(service['type'].upper())
+                except Exception: 
+                    pass
+                    
+                for characteristic in service['characteristics']:
+                    characteristic['type'] = characteristic['type'].upper()
+                    try:
+                         characteristic['type'] = CharacteristicsTypes.get_uuid(characteristic['type'].upper())
+                    except Exception: 
```

Comment:
Would it be better to just catch `KeyError` here?

Reply:
Sorry it's my one of first encounters with python, will fix as you suggest, seems logical.